uefi-test-runner/src/boot/misc.rs

```diff
@@ -112,7 +112,7 @@ fn test_install_protocol_interface(bt: &BootServices) {
             .expect("Failed to install protocol interface")
     };
 
-    let exists = bt
+    let _ = bt
```

Comment:
nit. this commit could be squashed with the previous one

Reply:
I need to learn git a bit better anyway, so I'll see what I can do about cleaning up the history a bit today.

Reply:
(y) Moving things within existing git commits is kinda hard. In the last months, I've spent some time improving my skills. The following workflow worked for me. Maybe it helps you as well, hopefully:

- 1. `git rebase --interactive HEAD~4` (for four commits)
- in the editor, mark all commits you want to change with `e`/`edit`
- run `cargo fmt` && `cargo clippy` plus `git add` all changes
- run `git rebase --continue`

Ideally, git should discard the "code style only commit" if you can pull these changes into earlier commits this way. Let me know what worked for you! :)

Reply:
That actually was a really easy way to understand how to split commits apart, thanks! I had no idea that git would detect and let you remove empty commits.